src/cljs/main/broadfcui/main.cljs

```diff
@@ -249,9 +233,14 @@
                :else [LoggedIn {:component component :make-props make-props}]))]]
          (footer/render-footer)
          (when (:showing-system-down-dialog? @state)
-           (render-system-status-dialog
-            (:maintenance-mode? @state)
-            #(swap! state dissoc :showing-system-down-dialog? :maintenance-mode?)))
+           (let [title (if (:maintenance-mode? @state) "Maintenance Mode"
+                                                       "Server Unavailable")
+                 msg (if (:maintenance-mode? @state) "FireCloud is currently undergoing planned maintenance. We should be back online shortly."
+                                                     "FireCloud service is temporarily unavailable.")]
+             (notifications/render-alert {:title title
+                                          :message msg
+                                          :link "http://status.firecloud.org/"
+                                          :cleared? false} nil)))
```

Comment:
in functional testing, this banner can get "stuck" visible. To reproduce using local UI + local orch:
1. start all services, healthy. Log in to the UI?. Navigate to workspace list. Notice all works.
2. shut down orch, navigate to Library using top nav. Notice banner.
3. start orch, navigate to Workspaces. Notice system is healthy but banner remains.

Reply:
Yes, this is an excellent observation.

Right now, the AJAX "health monitor" that flags the system as down has no functionality to mark it as being back up.

With the popups, this works fine because the user just closes the popup. Unfortunately, the banner will persist until the user refreshes the page.

It's not immediately obvious to me what condition we should check to make the banner go away (a single successful AJAX seems too simplistic/naive), will put some thought into it

Reply:
After thinking about this a bit, I decided to implement the last paragraph, above.

When Orch gets into a state where it returns 5xx codes (what we care about for the calling FC "down"), it just uniformly returns that until it's back up. A single Orch AJAX success is the right thing to check, I believe.